SECURITY.md

```diff
@@ -14,7 +14,7 @@ A "vulnerability in Mastodon" is a vulnerability in the code distributed through
 ## Supported Versions
 
 | Version | Supported        |
-| ------- | ---------------- |
+|---------|------------------|
```

Comment:
I think the linter prefers the version with spaces on edges of horizontal line here. All other changes look good.